routes/user.js

```diff
@@ -7,6 +7,21 @@ const bcrypt = require("bcrypt");
 
 const jwt = require("jsonwebtoken");
 
+const authenticateToken = (req, res, next) => {
+  const authHeader = req.headers["authorization"];
+  const token = authHeader && authHeader.split(" ")[1];
+
+  if (token == null) {
+    return res.status(401).send("No token registered");
+  }
+
+  jwt.verify(token, process.env.ACCESS_TOKEN_SECRET, (err, user) => {
+    if (err) return res.status(403).send("The token is unvalid");
```

Comment:
```suggestion
    if (err) return res.status(403).send("The token is invalid");
```
Not a major thing, but a more common word is invalid